openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -160,8 +164,7 @@ const Carousel = {
 
                 if (!loadMore.locked && !loadMore.allDone && isOn2ndLastPage) {
                     loadMore.locked = true; // lock for critical section
-                    slick.addSlide('<div class="carousel__item carousel__loading-end">Loading...</div>');
-
+                    slick.addSlide('<div class="carousel__item carousel__loading-end">i18nValues[loading"]</div>');
```

Comment:
```suggestion
                    slick.addSlide(`<div class="carousel__item carousel__loading-end">${i18nValues[loading"]}</div>`);
```
Use template literals for string interpolation.